### PR TITLE
feat: divider dot component

### PR DESCRIPTION
**Pull request by tallidin**
# Description
Divider dot component

image

# Links
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-292468

### CODE REVIEW

File: src/components/DividerDot/DividerDot.stories.docs.mdx [hunk 1]
```diff
@@ -0,0 +1 @@
+The `<DividerDot />` component.
```
Comment (cscheffauer) [inline src/components/DividerDot/DividerDot.stories.docs.mdx:1]: Having a documentation here would make sense I think :)

File: src/components/DividerDot/DividerDot.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    expanded: true,
+    docs: {
+      page: DocumentationPage(Documentation, StyleDocs),
+    },
+  },
+};
+
+const Example = Template<DividerDotProps>(DividerDot).bind({});
+
+Example.argTypes = { ...argTypes };
```
Comment (InteractiveTimmy) [inline src/components/DividerDot/DividerDot.stories.tsx:22]: Can a story be inserted that shows this specific use case of this component? i.e. a `Common` story that includes text being divided by the `<DividerDot />`?

**Review by InteractiveTimmy — CHANGES_REQUESTED** [review]
Looks good! Just need a common story to demonstrate the usage.

**Review by cscheffauer — APPROVED** [review]

File: src/components/DividerDot/DividerDot.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  return (
+    <>
+      <Text type="body-secondary" style={{ display: 'inline' }}>
+        first part of text
+      </Text>
+      <DividerDot />
+      <Text type="body-secondary">second part of text</Text>
+    </>
+  );
+};
```
Comment (InteractiveTimmy) [inline src/components/DividerDot/DividerDot.stories.tsx:36]: Thanks for adding this common example! approved!

**Review by InteractiveTimmy — APPROVED** [review]
LGTM!

**Pull request merged**